fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/expression/PredicateAdapter.java

```diff
@@ -0,0 +1,344 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.expression;
+
+import com.ibm.fhir.database.utils.query.node.BindMarkerNode;
+import com.ibm.fhir.database.utils.query.node.ExpNode;
+import com.ibm.fhir.database.utils.query.node.PredicateParser;
+
+/**
+ * An adapter used to build (simple) query predicates
+ */
+public class PredicateAdapter {
```

Comment:
What is the difference and reason for the duplication between BaseWhereAdapter and  this?   Are they serving the same purpose?

Reply:
Refactored. Deleted